doc/source/train/common/torch-configure-run.rst

```diff
@@ -1,10 +1,10 @@
-Configure scale and GPUs
-------------------------
+Configure scale and resources (CPU, GPU, and other accelerators)
```

Comment:
@angelinalg do you think this ends up being too long of a header? Any suggestions on shortening it while keeping the same meaning?